text/0003-multisig-minting/0003-multisig-minting.md

```diff
@@ -32,14 +32,16 @@ Adding a simple rule for multi-signature scripts that blocks should be signed by
 
 ## Advantages
 
-* More secure minting.
-* Permanent outsourcing of minting is not possible, every mint needs to be pre-signed by the holder.
-* Minimal protocol changes.
+* Enabling multiple-signature addresses to mint. While the coinstake would need to be signed by the required number of signatures, the block would only need to be signed by any one member of the multi-sig regardless of the number of signatures required for a normal transaction. 
+* Allowing users to only access their wallets once every maturation period (30 days in the current protocol) while still minting continuously, instead of having to give continuous wallet access to their clients.
+* Allowing mint pools to exist, though this will require third party infrastructure to navigate the periodic minting process.
 
 ## Drawbacks
 
-* Hard fork
+* Dependency on other RFCs with drawbacks.
+* The convention of using just one of the multi-sig keys to sign the block must be changed if alternatives are prefered in the future.
```

Comment:
Why? I don't understand this point.
To me it just looks like a non-argument that sais: "if you want to revert this RFC, you have to revert it".

Reply:
It is choosing a convention.  Choosing a convention always has at least the disadvantage of that once chosen you have to unchoose it if you want something else.

Reply:
If you want to owner to decide, you a different script type with possibly a new OP_CODE. This is still possible without reverting this RFC.
We cannot encode extra information into a standard multisig script, therefore it is needed to fix the number of required signatures in the protocol for minting such a script.
Also be aware that current block format does not allow multiple block signatures and that passing around a block for signatures after finding stake increases the chance of orphaning.

Reply:
I agree that this is not practically a disadvantage, or at least it is trivial enough to ignore it.  Id be fine removing this bullet point.